Sources/Controllers/Help/MenuHelpDataService.swift

```diff
@@ -204,7 +205,7 @@ final class MenuHelpDataService: NSObject, XMLParserDelegate {
     
     private func getArticlePropertyName(from url: String) -> String {
         var propertyName = url.lowercased().replacingOccurrences(of: kOsmAndUserBaseURL, with: "")
-        propertyName = propertyName.replacingOccurrences(of: "-", with: "_").replacingOccurrences(of: "/", with: "_")
+        propertyName = propertyName.replacingOccurrences(of: "-", with: "_").replacingOccurrences(of: "/", with: "_").replacingOccurrences(of: " ", with: "_")
```

Comment:
Many replacing occurrences. Create a CharacterSet with characters ( CharacterSet(charactersIn: "-/ ")) for use with replacingOccurrences